Three/include/CGAL/Three/Scene_group_item.h

```diff
@@ -196,7 +196,7 @@ public :
     //!
     //! Only returns children that have this item as a parent.
     //! Children of these children are not returned.
-    QList<Scene_interface::Item_id> getChildren() const {return children;}
+    QList<Scene_interface::Item_id> getChildren() const {return *children;}
```

Comment:
You will still have the warning about the DLL export stuff.

Reply:
The only warning I have left in 
```
C:\Cygwin\home\afabri\cgal\Three\include\CGAL/Three/Scene_group_item.h(263,63): warning C4251: 'CGAL::Three::Scene_group_item::children': class 'std::unique_ptr<QList<int>,std::default_delete<QList<int>>>' needs to have dll-interface to be used by clients of class 'CGAL::Three::Scene_group_item' [C:\Cygwin\home\afabri\cgal\Polyhedron\demo\build\scene
_c3t3_item.vcxproj]
```
So no, it should not be a unique_ptr, what I did was right.

Reply:
I do not understand why this internal data member is part of the DLL boundary.